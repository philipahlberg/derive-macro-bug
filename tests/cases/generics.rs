use derive_macro_bug::Macro;

#[derive(Macro)]
struct Foo<T> {
    bar: T,
}

fn main() {}
