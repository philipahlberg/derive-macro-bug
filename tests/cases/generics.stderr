error: oops
 --> $DIR/generics.rs:4:11
  |
4 | struct Foo<T> {
  |           ^
